.github/workflows/caching-build.yml

```diff
@@ -14,8 +14,8 @@ jobs:
       - run: sudo apt update
       - run: sudo apt install -y git --no-install-recommends
       # get latest head commit of sched_ext for-next
-      - run: echo "SCHED_EXT_KERNEL_COMMIT=$(git ls-remote https://git.kernel.org/pub/scm/linux/kernel/git/tj/sched_ext.git heads/for-next | awk '{print $1}')" >> $GITHUB_ENV
 
+      - run: echo "SCHED_EXT_KERNEL_COMMIT=21f4c19" >> $GITHUB_ENV
```

Comment:
does this need updated on each update?

Reply:
yes for each kernel commit/branch/repo triple you want to backport against.

Reply:
ahh for clarity, those are params are on the kernel sources, not the library repo. the library repo commit used is that which the backport branch is rebased upon (i.e. this stacks on that).